Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/ServerGameNetPortal.cs

```diff
@@ -230,6 +234,16 @@ public void SendServerToClientSetDisconnectReason(ulong clientID, ConnectStatus
             writer.WriteValueSafe(status);
             NetworkManager.Singleton.CustomMessagingManager.SendNamedMessage(nameof(ClientGameNetPortal.ReceiveServerToClientSetDisconnectReason_CustomMessage), clientID, writer);
         }
+        /// <summary>
+        /// Sends a DisconnectReason to all connected clients. This should only be done on the server, prior to disconnecting the clients.
+        /// </summary>
+        /// <param name="status"> The reason for the upcoming disconnect.</param>
+        public void SendServerToAllClientsSetDisconnectReason(ConnectStatus status)

```

Comment:
I thought we already had this?

Reply:
We had this for sending it to a specific client. Adding this is simpler than looping through ConnectedClientIds to call the method we already had